chrome_manifest_v3/model.js

```
@@ -1,784 +1,764 @@
+              (value || '').toString().length +
+              ')'
+          );
+        });
+        return keysAndValues.sort().join(' ');
+      };
+
+      const dict_k = { cardId: this.cardId || '' };
+
+      let method = upload1.method || 'post';
+      let property = this.parent.parent.replacer(upload1.property, dict_k);
+      delete upload1.method;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)